.travis.yml

```diff
@@ -2,14 +2,7 @@ language: julia
 os:
   - linux
 julia:
-  - 0.6
+  - 0.7
   - nightly
 notifications:
   email: false
-#script: # the default script is equivalent to the following
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia -e 'Pkg.clone(pwd()); Pkg.build("QuadGK"); Pkg.test("QuadGK"; coverage=true)';
-after_success:
-  - julia -e 'cd(Pkg.dir("QuadGK")); Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))';
```

Comment:
Don't we still want documentation and coverage?

Reply:
What's the way to do this in 0.7?   e.g. `Pkg.dir` is gone.

Reply:
`Pkg.dir("Package")` is still available, but deprecated. `Pkg.dir()` is gone.

Reply:
Okay, reverted for now, but 0.7 will need a non-deprecated solution here.